src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -911,6 +929,58 @@ struct data_retention
 
 };
 
+struct mailbox
+{
+    using result_type = boost::any;
+
+    static xcl_mailbox
+        init_mailbox_info(const xrt_core::device* dev)
+    {
+        xcl_mailbox info = { 0 };
+        userpf::get_mailbox_info(dev->get_user_handle(), &info);
```

Comment:
Too bad the API is not simple as
`return userpf::get_mailbox_info(dev->get_user_handle())`

Reply:
followed the same syntax of the code from the same file "device_windows.cpp"